backend/api/Areas/Admin/Controllers/UserController.cs

```diff
@@ -54,9 +55,10 @@ public UserController(ILogger<UserController> logger, IConfiguration configurati
         /// <param name="page"></param>
         /// <param name="quantity"></param>
         /// <param name="sort"></param>
+        /// <param name="userId"></param>
         /// <returns>Paged object with an array of users.</returns>
         [HttpGet("/api/admin/users")]
-        public IActionResult GetUsers(int page = 1, int quantity = 10, string sort = null) // TODO: sort and filter.
+        public IActionResult GetUsers(int page = 1, int quantity = 10, string sort = null, Guid? userId = null) // TODO: sort and filter.
```

Comment:
It looks like you're using the same endpoint for either fetching one user or fetching many.  If so I'd recommend you create a new endpoint for the single user.

Reply:
If you are using the `userId` as a way to fetch the current user's Id, then just use the context `User.GetUserId()`.

Reply:
This is probably confusing, but I'm using that user id as the 'logged in user'. So I'm not getting a single user based on that user id, I'm actually getting all users with agencies 'belonging' to that user id.

Reply:
Ah in that case you don't need it.  If it's the logged in user, we have that in `User.GetUserId()`.